notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Since there are different variations of Diffusion model implementation, it would be better to explain which model this notebook will implement. Maybe DDPM?</p><p>https://arxiv.org/abs/2006.11239</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done.</p>

Reply:
<p>This link seems to be DDIM, not DDPM.</p><p>And the algorithm is basically based on DDPM (except for scheduling section?)</p>